Translated/UI/ui_mainmode.csv

```diff
@@ -1590,11 +1590,11 @@ Name#0,"""Character Name"""
 Guild#0,"""Team Name"""
 NoGuild#0,"""Teamless"""
 HunterExplanation#0,"""<yellow>Close Ranged Class<c><br>A class whose value lies in close-range combat. Has high HP and defense, and is excellent at fighting single targets."""
-RangerExplanation#0,"""<yellow>中距離戦闘向きのクラス<c><br>射撃武器により、エネミーと一定の距離を置いて戦う。<br>特殊弾で、他のプレイヤーを支援することもできる。"""
-ForceExplanation#0,"""<yellow>遠距離戦闘向きのクラス<c><br>遠距離から、“テクニック”でエネミーを攻撃する。<br>多彩なテクニックで、エネミーの弱点属性を巧みに狙う。"""
-FighterExplanation#0,"""<yellow>超近距離戦闘向きのクラス<c><br>エネミーと肉薄する距離での戦闘を得意とする。<br>防御力を犠牲にした強力な猛攻撃で、エネミーを粉砕する。"""
-GunnerExplanation#0,"""<yellow>近距離戦闘向きのクラス<c><br>射撃武器による近接戦闘を得意とする。<br>すさまじい弾幕と優れた機動力で、エネミーを翻弄する。"""
-TecherExplanation#0,"""<yellow>近距離および遠距離戦闘向きのクラス<c><br>“テクニック”に長け、他のプレイヤーを支援する。<br>一時的な能力強化や回復などで、戦闘を有利に進める。"""
+RangerExplanation#0,"""<yellow>Medium Ranged Class<c><br>Fight while keeping distance from enemies with a ranged weapon.<br>You can also support other players with special bullets."""
+ForceExplanation#0,"""<yellow>Long Ranged Class<c><br>Attack enemies from long ranges using \""Techniques\"".<br>Harass enemies' elemental weaknesses by using various Techniques."""
+FighterExplanation#0,"""<yellow>Ultra-close Ranged Class<c><br>Close in and take the fight personally with enemies.<br>Crush your enemies with powerful, fierce attacks at the expense of your defense."""
+GunnerExplanation#0,"""<yellow>Close Ranged Class<c><br>Good at close-quarters combat with a ranged weapon.<br>Unleash fierce barrages and utilize your mobility to bamboozle enemies."""
+TecherExplanation#0,"""<yellow>Close/Long Ranged Class<c><br>Support other players using \""Techniques\"".<br>Shift the battle in your favor by buffing and healing your teammates."""
```

Comment:
Don't need quotes on Techniques. Also, "Shift the battle *to* your favour" would probably read a bit better.

Reply:
I googled it up and it seems that the correct grammar is indeed _in_ your favor, so I'll leave this one unchanged.